src/models/game_session.ts

```diff
@@ -24,6 +25,10 @@ const LAST_PLAYED_SONG_QUEUE_SIZE = 10;
 export default class GameSession {
     private readonly startedAt: number;
 
+    public readonly eliminationMode: boolean;
```

Comment:
This should probably be of type `gameType`, the enum you created instead of specific boolean